### ISSUE TITLE
infretisrun when restart.toml exists

### ISSUE THREAD

**Issue by freschavo** [infretis/infretis#129]
No warning when (erroneously, without proper precaution) starting an infretisrun 
`infretisrun -i infretis.toml`
in a directory with an existing restart.toml.
In this case, existing restart.toml is overwritten.
Also potentially overwrites load, continues writing sim.log etc as if from beginning.

**Comment by lukasbaldauf** [comment]
We could just raise an Error on startup if a restart.toml exists, wdyt @dz24? Right now we compare the infretis.toml and restart.toml, but somewhere along the line it stopped working.

**Comment by dz24** [comment]
Yes, we could enforce the policy that one should always run a "restart.toml" file if it is available, and only other toml files are accepted if it is not present.

I was thinking if it would be better to introduce this "dummyproof" feature, but then it would "lessen" the flexibility in the case that the user does not care about restart.toml and just want to run some other toml file. But overall that is a minor issue and the user can easily fix it so I would say we can just indeed go prioritizing the restart toml file

**Comment by lukasbaldauf** [comment]
+1 on that ...
